src/simulator.py

```diff
@@ -0,0 +1,19 @@
+from sim.UI.view import View
+from sim.ModelClass import Model
+from sim.controller.ControllerClass import Controller
+
+def main():
+    # Create Model
+    model = Model()
+
+    # Create Controller
+    controller = Controller(model)
+
+    # Create View
+    view = View(controller)
```

Comment:
May want to keep an eye on not threading the view. If workers are external, should be alright but if internal may cause freezing in GUI

Reply:
View should not be threaded